images/win/scripts/Installers/Install-Chrome.ps1

```diff
@@ -3,24 +3,40 @@
 ##  Desc:  Install Google Chrome
 ################################################################################
 
-Import-Module -Name ImageHelpers -Force
+Import-Module -Name ImageHelpers -Force;
 
-$temp_install_dir = 'C:\Windows\Installer'
-New-Item -Path $temp_install_dir -ItemType Directory -Force
+$ChromeInstallerFile = "chrome_installer.exe";
+$ChromeInstallerUri = "https://dl.google.com/chrome/install/375.126/${ChromeInstallerFile}";
```

Comment:
is this a static url for a particular version or is it where the latest version will be available?

If it is static what is our cadence for updating and who'll be taking responsibility for periodically updating it?

Reply:
Yep, this link points to the latest stable version so we don't need to maintain it manually in future